test/DebuggerCommon/argument_disp.js.dbg.baseline

```diff
@@ -1,6 +1,7 @@
 [
   {
     "this": {
+      "chWriteTraceEvent": "function <large string>",

```

Comment:
>chWriteTraceEvent [](start = 7, length = 17)

Instead of modifying all baselines add "chWriteTraceEvent" to filterLog in bin\ch\DbgController.js. That should exclude chWriteTraceEvent from all baselines.